import sys

def report_error(message):
    print(message)
    sys.exit(0)
